src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -1,67 +1,66 @@
 // @debug $text-input-group; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
+:where(:root),
+:where(.#{$text-input-group}) {
+  --#{$text-input-group}--BackgroundColor: var(--pf-t--global--background--color--control--default);
+  --#{$text-input-group}--BorderColor: var(--pf-t--global--border--color--default);
+  --#{$text-input-group}--BorderWidth: var(--pf-t--global--border--width--control--default);
+  --#{$text-input-group}--BorderRadius: var(--pf-t--global--border--radius--small);
 
-.#{$text-input-group} {
-  --#{$text-input-group}--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
-
-  // Base border
-  --#{$text-input-group}__text--before--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$text-input-group}__text--before--BorderColor: var(--#{$pf-global}--BorderColor--300);
-
-  // Bottom border
-  --#{$text-input-group}__text--after--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$text-input-group}__text--after--BorderBottomColor: var(--#{$pf-global}--BorderColor--200);
-  --#{$text-input-group}--hover__text--after--BorderBottomColor: var(--#{$pf-global}--primary-color--100);
-  --#{$text-input-group}__text--focus-within--after--BorderBottomWidth: var(--#{$pf-global}--BorderWidth--md);
-  --#{$text-input-group}__text--focus-within--after--BorderBottomColor: var(--#{$pf-global}--primary-color--100);
+  // Border hover
+  --#{$text-input-group}--m-hover--BorderColor: var(--pf-t--global--border--color--hover);
 
   // Main
-  --#{$text-input-group}__main--first-child--not--text-input--MarginLeft: var(--#{$pf-global}--spacer--sm);
-  --#{$text-input-group}__main--m-icon__text-input--PaddingLeft: var(--#{$pf-global}--spacer--xl);
-  --#{$text-input-group}__main--RowGap: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}__main--ColumnGap: var(--#{$pf-global}--spacer--sm);
+  --#{$text-input-group}__main--first-child--not--text-input--MarginLeft: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__main--m-icon__text-input--PaddingLeft: var(--pf-t--global--spacer--xl);
+  --#{$text-input-group}__main--RowGap: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__main--ColumnGap: var(--pf-t--global--spacer--xs);
 
   // Chip group
-  --#{$text-input-group}--c-chip-group__main--PaddingTop: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}--c-chip-group__main--PaddingRight: var(--#{$pf-global}--spacer--xs);
-  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--#{$pf-global}--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingTop: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingRight: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-chip-group__main--PaddingBottom: var(--pf-t--global--spacer--xs);
 
   // Text input
   --#{$text-input-group}__text-input--PaddingTop: var(--#{$pf-global}--spacer--form-element);
```

Comment:
Any reason you wouldn't go ahead and update this and the bottom padding to use the small spacer? We're doing that in a bunch of components, and that would make the size of this component correct. We have an issue to go back and add the semantic spacer later, which we can add here.